packages/material-ui/scripts/rollup.config.js

```diff
@@ -43,17 +41,12 @@ export default [
     external: Object.keys(globals),
     plugins: [
       nodeResolve(),
-      babel(getBabelOptions()),
-      commonjs(getCommonjsOptions()),
+      babel(babelOptions),
+      commonjs(commonjsOptions),
+      nodeGlobals(),
       replace({ 'process.env.NODE_ENV': JSON.stringify('production') }),
       sizeSnapshot(),
       uglify(),
     ],
   },
-  {
```

Comment:
Does this bundle block you somehow? I still want to work on treeshakability. Just didn't have enough time last weeks.

Reply:
This is dead code now. I have removed `sizeSnapshot` in #11797.